main-app/app/scripts/login/login-service.js

```diff
@@ -0,0 +1,7 @@
+(function () {
+    'use strict';
+    angular.module('Tombola.BingoClient.Login')
```

Comment:
Ignore comment above....
Assuming this to because this PR is a first cut.
Best to leave a //TODO comment explaining WIP